contributors/design-proposals/storage/pv-to-rbd-mapping.md

```diff
@@ -0,0 +1,116 @@
+# RBD Volume to PV Mapping 
+
+Authors: krmayankk@
+
+### Problem
+
+The RBD Dynamic Provisioner currently generates rbd volume names which are random. 
+The current implementation generates a UUID and the rbd image name becomes 
+image := fmt.Sprintf("kubernetes-dynamic-pvc-%s", uuid.NewUUID()). This RBD image 
+name is stored in the PV. The PV also has a reference to the PVC to which it binds. 
+The problem with this approach is that if there is a catastrophic etcd data loss 
```

Comment:
I am just curious, if there is `catastrophic etcd data loss`, do you think with all other info lost, having deterministic volume name would help?

Reply:
I have had use cases where you might want to import the volume in a new cluster and with out the meta-data details we don't know the origin of the volume.